gemfiles/redis-activesupport.gemfile

```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gem "redis-activesupport"
+
+gemspec path: "#{__dir__}/.."
```

Comment:
Current order matters here - must load our gem after redis-activesupport in order to enable the monkey patch. Need to figure out a better way to set this up.